package.json

```diff
@@ -3,13 +3,16 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/core": "^10.0.10",
```

Comment:
Can you detail why we need `emotion` and why it's the best alternative?

Reply:
@jeluard -- Storybook Theming under the hood relies on [`emotion`](https://storybook.js.org/docs/configurations/theming/#using-the-theme-for-addon-authors). There seems to be an issue with the version of `@emotion@10.0.7` in `storybook@5.0.5`, hence I had to add the it to the `package.json` explicitly.

Also, I've re-used most of the code from CircleCI as well as followed there setup process as I've really liked the way things are done there. I've compared `react-emotion` vs `styled-components` and there doesn't seem to be much difference between both of them.

Combining all the above points, I didn't see a reason to not use `@emotion` for the ease of development :)